server/models/Message.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require("mongoose");
+
+const messageSchema = new mongoose.Schema(
+  {
+    conversationId: {
```

Comment:
For this field and the one below aren't we referencing the relevant models? :)

Reply:
Thanks for pointing that out! The added the correct refs to the respective models.